content/en/data_streams/troubleshooting.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Just noting that this page isn't being linked to from any other docs, and it also doesn't appear in the left-nav sidebar (in the `menus.en.yaml` file). Do we want to add it there?

Reply:
Yes please! Can we follow what other products do? https://a.cl.ly/L1uDEDwp

Similarly on the docs landing page (https://docs.datadoghq.com/), can we add a Data Streams Monitoring tile?

Reply:
Sounds good on adding the Troubleshooting guide to the left nav.
For the docs landing page, this isn't urgent and I can submit a ticket to the Docs Jira later

Everything in this PR has to be merged today since today is the GA date, the sooner the better though